README.md

```diff
@@ -40,44 +40,44 @@ public class ColorChooserThatDoesntLikeGreen : ReactiveObject
   // Declaring a read/write property
   //
 
-  byte _Red;
+  byte red;
   public byte Red {
-    get { return _Red; }
-    set { this.RaiseAndSetIfChanged(value); }
+    get { return red; }
+    set { this.RaiseAndSetIfChanged(ref red, value); }
   }
 
-  byte _Green;
+  byte green;
   public byte Green {
-    get { return _Green; }
-    set { this.RaiseAndSetIfChanged(value); }
+    get { return green; }
+    set { this.RaiseAndSetIfChanged(ref green, value); }
   }
 
-  byte _Blue;
+  byte blue;
   public byte Blue {
-    get { return _Blue; }
-    set { this.RaiseAndSetIfChanged(value); }
+    get { return blue; }
+    set { this.RaiseAndSetIfChanged(ref blue, value); }
   }
 
   //
   // Declaring a Property that's based on an Observable
   // 
 
-  ObservableAsPropertyHelper<Color> _Color;
+  ObservableAsPropertyHelper<Color> color;
   public Color Color {
-    get { return _Color.Value; }
+    get { return color.Value; }
   }
 
-  ReactiveCommand OkButton { get; protected set; }
+  public ReactiveCommand<object> OkButton { get; protected set; }
 
   public ColorChooserThatDoesntLikeGreen()
   {
-    var finalColor = this.WhenAny(x => x.Red, x => x.Green, x => x.Blue, 
-        (r,g,b) => Color.FromRGB(r.Value, g.Value, b.Value));
+    var finalColor = this.WhenAny(x => x.Red, x => x.Green, x => x.Blue,
+      (r,g,b) => Color.FromArgb(r.Value, g.Value, b.Value));
```

Comment:
There is an overload without the alpha parameter.

https://msdn.microsoft.com/en-us/library/cce5h557%28v=vs.110%29.aspx